### DISCUSSION TITLE
Karavan in Docker - Setup error

### DISCUSSION THREAD

**Discussion by majdiAlKotamy** [apache/camel-karavan#1209] [Q&A] [answered]
Hi @mgubaidullin,

I Have an issue wanna share it with you, when I try to install Karavan in Docker.
Take a look please on the attached screenshot, knowing I am using the same docker compose file tat existeed in the repo, and I have stored the constants values on ubuntu env like 

```
KARAVAN_GIT_REPOSITORY=https://github.com/apache/camel-karavan
KARAVAN_GIT_USERNAME=majdi
KARAVAN_GIT_PASSWORD=*******
REGISTRY_HTTP_SECRET=*******
```

<img width="959" alt="Capture1" src="https://github.com/apache/camel-karavan/assets/62914344/58113d27-e3c5-4699-b65b-5ecc696834c5">

<img width="959" alt="Capture2" src="https://github.com/apache/camel-karavan/assets/62914344/0c6046c9-9989-49df-aeb6-f6a46986ace9">

looking to your assitance 
Thanks in advance

**Comment by mgubaidullin** [comment, marked as answer]
1. First three lines on your screenshot indicates that you did not configure env variables correct
2. KARAVAN_GIT_REPOSITORY should be YOUR repository to store YOUR projects
